README.md

```diff
@@ -74,8 +75,9 @@ android {
         javaCompileOptions {
             annotationProcessorOptions {
                 arguments = [
-                    stagGeneratedPackageName: 'com.vimeo.sample.stag.generated',
-                    stagDebug               : 'true'
+                    stagAssumeHungarianNotation: 'true'
+                    stagGeneratedPackageName   : 'com.vimeo.sample.stag.generated',
+                    stagDebug                  : 'true'
```

Comment:
Good points. Looks like processor args is the best bet - maybe with class level overrides as a future feature. And auto detection would be a pretty sweet reach goal (if there was a way to guarantee it's correct).